lib/hammer_cli_katello/repository.rb

```diff
@@ -83,10 +84,14 @@ class InfoCommand < HammerCLIKatello::InfoCommand
       end
 
       def extend_data(data)
-        if data["content_type"] == "yum" && data["gpg_key"]
-          data["gpg_key_name"] = data["gpg_key"]["name"]
+        if data["content_type"] == "yum"
+          if data["ignorable_content"]
+            data["_ignorable_content"] = data["ignorable_content"].join(",")
```

Comment:
I don't mind overwriting the data coming back from the API. No need to make a new key imo. Also, [hammer-cli-foreman does it](https://github.com/theforeman/hammer-cli-foreman/blob/170e7d8aac5b7d05d76ce0379cfde4ff2de38f65/lib/hammer_cli_foreman/settings.rb#L18).

Reply:
> It looks good, just one comment. I'd like you to include an updated apipie cache since you changed the API in Katello.

Not needed. I see the updated API json merged already .